server/methods/browseChannels.js

```diff
@@ -4,7 +4,7 @@ import { hasPermission } from '../../app/authorization';
 import { Rooms, Users } from '../../app/models';
 import s from 'underscore.string';
 
-import { Federation } from '../../app/federation';
+import { Federation, PeerClient } from '../../app/federation';
```

Comment:
```suggestion
import { Federation } from '../../app/federation';
import peerClient from '../../app/federation/server/peerClient';
```

as per [above](https://github.com/RocketChat/Rocket.Chat/pull/13836/files#diff-89cf71887f15469b951ac142c744f27eR25) suggestion